app/app/utils.py

```diff
@@ -161,6 +162,7 @@ def setup_lang(request, user):
         DoesNotExist: The exception is raised if no profile is found for the specified handle.
 
     """
+    from dashboard.models import Profile
```

Comment:
Why did you move this import within the function from the top of the file?

Reply:
Aah. i need to remove this. I was previously importing util in dashboard and it was causing a recursive importing, so. Thanks for pointing out.

Reply:
How much time do you think it would take to refactor out the recursive import? Imports within a function aren't terribly bad, but I think for webservers they can really affect performance.

Reply:
It will take a lot of adjustments in a lot of places. If it's not too much bad for the performance, I think we can do it in a later adjustment.